credoai/metrics/metric_constants.py

```diff
@@ -62,3 +62,19 @@
     "demographic_parity_ratio",
     "demographic_parity_difference"
 ]
+
+# REGRESSION METRICS
+REGRESSION_FUNCTIONS = {
+    'explained_variance_score': sk_metrics.explained_variance_score,
+    'max_error': sk_metrics.max_error,
+    'mean_absolute_error': sk_metrics.mean_absolute_error,
+    'mean_squared_error': sk_metrics.mean_squared_error,
```

Comment:
Can you add an "RMSE" metric? I think it's common enough we'd want it. I think using `partial` (from `functools`) on [MSE](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html), will work (just set "squared=False")

Reply:
Added
`'root_mean_squared_error': partial(sk_metrics.mean_squared_error, squared=False)`